Formula/fuseki.rb

```diff
@@ -1,27 +1,25 @@
 class Fuseki < Formula
   desc "SPARQL server"
   homepage "https://jena.apache.org/documentation/serving_data/"
-  url "https://archive.apache.org/dist/jena/binaries/jena-fuseki1-1.1.2-distribution.tar.gz"
-  version "1.1.2"
-  sha256 "78bd92b4e32f9e918d89946d11aed9789416f4058b127af60b251b4a8636b5f0"
+  url "https://archive.apache.org/dist/jena/binaries/jena-fuseki1-1.4.0-distribution.tar.gz"
+  version "1.4.0"
+  sha256 "8f0cfa13d9a94df9dfb5a424177a0d3d9873b605e0ae610ba6e5f3d30f06f9bf"
 
   bottle :unneeded
+  plist_options :manual => "fuseki start"
```

Comment:
@imgarylai Please move `plist_options :manual => "fuseki start"` below the `def install` method, thanks.